src/Entity/Auth/UserProvider.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace App\Entity\Auth;
+
+use Cocur\Slugify\SlugifyInterface;
+use Doctrine\ORM\EntityManagerInterface;
+use HWI\Bundle\OAuthBundle\OAuth\Response\UserResponseInterface;
+use HWI\Bundle\OAuthBundle\Security\Core\User\OAuthAwareUserProviderInterface;
+use Symfony\Component\Security\Core\Exception\UnsupportedUserException;
+use Symfony\Component\Security\Core\Exception\UsernameNotFoundException;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Component\Security\Core\User\UserProviderInterface;
+
+class UserProvider implements UserProviderInterface, OAuthAwareUserProviderInterface
+{
+    /**
+     * @var EntityManagerInterface
+     */
+    protected $entityManager;
```

Comment:
`private` ?

Reply:
Since we are trying to remove this and using only repositories, I'll mark this comment as `Resolved`